src/context.rs

```diff
@@ -46,24 +50,46 @@ pub static tx_size_wide_log2: [usize; TX_SIZES_ALL] =
     [2, 3, 4, 5, 2, 3, 3, 4, 4, 5, 2, 4, 3, 5];
 pub static tx_size_high_log2: [usize; TX_SIZES_ALL] =
     [2, 3, 4, 5, 3, 2, 4, 3, 5, 4, 4, 2, 5, 3];
+// Transform block width in pixels
+pub static tx_size_wide: [usize; TX_SIZES_ALL] =
```

Comment:
I want to add width() and height() to the enum like bsize has, but that can wait.